measurements/api/measurements.py

```diff
@@ -539,6 +539,18 @@ def list_measurements(
         fpq = fpq.filter(Fastpath.test_name == test_name)
     if input_:
         fpq = fpq.filter(Fastpath.input == input_)
+    elif domain:
+        if test_name in [None, "web_connectivity", "http_requests"]:
+            fpq = fpq.filter(
+                or_(
+                    Fastpath.input.startswith(domain),
+                    Fastpath.input.startswith("http://" + domain),
+                    Fastpath.input.startswith("https://" + domain),
+                )
+            )
+        else:
+            fpq = fpq.filter(Fastpath.input.startswith(domain))
```

Comment:
5Looks like this code path is not used by Explorer: domain is set while test_name is not in [None, "web_connectivity", "http_requests"]

Should it run or return error?

Reply:
It should run (I think), because you want to be be able to filter for results of tests like the `dns_consistency` or `tcp_connect` tests which have as a target just a bare hostname